data/common.yaml

```diff
@@ -41,6 +41,7 @@ mcollective::server: true
 mcollective::client: false
 mcollective::purge: false
 mcollective::facts_refresh_interval: 10
+mcollective::facts_pidfile: "/var/run/mcollective-facts_refresh.pid"
```

Comment:
this should be in `/var/run/puppetlabs` - looks like the last thing to change sorry for the repeated review updates